infrastructure/ansible/roles/nginx/tasks/main.yml

```diff
@@ -4,6 +4,12 @@
     name: nginx
     state: present
 
+- name: create SSL keys if needed
+  become: true
+  command: openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout {{ nginx_ssl_certificate_key }} -out {{ nginx_ssl_certificate }} -batch
```

Comment:
Not sure about the letsencrypt config, but at least on prod I prefer to generate 4096 bit keys, might make sense to keep that consistent

Reply:
I udated the command, I realized that cerbot isn't part of ansible so far, so nevermind.
Just something to keep in mind.